spec/helpers/games_helper_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the GamesHelper. For example:
+#
+# describe GamesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe GamesHelper, type: :helper do
+  describe '#draw_piece' do
+    it 'returns nil when there is no piece at the given location' do
+      game = FactoryGirl.create(:game)
```

Comment:
Tests look great.

One thing you can do to DRY this up a bit.

After line 14, add `let(:game) { FactoryGirl.create(:game) }`

Then you can remove `game = FactoryGirl.create(:game)` everywhere.